config/sidekiq.yml

```diff
@@ -1,2 +1,8 @@
 ---
 :concurrency: 1
+:logfile: ./log/sidekiq.json.log
+:timeout: 4
+:max_retries: 9
```

Comment:
I see the above two values are [taken from email-alert-api](https://github.com/alphagov/email-alert-api/blob/df501d3edb7185ce35ab5592489765ec3699dae5/config/sidekiq.yml#L5-L6).

The use case for Locations API is different - if a postcode update fails, it's less important than if an email fails to be sent. I think it's reasonable to keep retries low. The [default is 25](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry), which is too much; I think we can actually set this to `1`.

The [default timeout is 25s](https://github.com/mperham/sidekiq/blob/152c3d8104295afc5b34ed9c190eaa85806b722f/lib/sidekiq.rb#L26), and again it seems reasonable to cut this value down. 4s seems fair 👍